bin/rwahs

```diff
@@ -87,7 +88,7 @@ function getRemoteCommand {
         source "${toolsDir}/config/${programName}/env/${environment}"
 
         # Output the command to run; this is captured by the calling function.
-        echo ssh "${user}@${host}" "'${targetRoot}/${targetParent}/bin/${programName} $@'"
+        echo ssh "${user}@${host}" "${targetRoot}/${targetParent}/current/bin/${programName} ${task} $@"
```

Comment:
I forgot that the `task` was already `shift`ed out, so it needs to be passed in explicitly.